Geometry/RPCGeometryBuilder/src/RPCGeometryBuilderFromCondDB.cc

```diff
@@ -57,26 +57,23 @@ RPCGeometry* RPCGeometryBuilderFromCondDB::build(const RecoIdealGeometry& rgeo)
     RPCRollSpecs* rollspecs = nullptr;
     Bounds* bounds = nullptr;
 
-    //    if (dpar.size()==4){
     if (rgeo.shapeEnd(id) - shapeStart == 4) {
       const float width = *(shapeStart + 0) / cm;
       const float length = *(shapeStart + 1) / cm;
       const float thickness = *(shapeStart + 2) / cm;
       const float nstrip = *(shapeStart + 3);
-      //RectangularPlaneBounds*
+
       bounds = new RectangularPlaneBounds(width, length, thickness);
       const std::vector<float> pars = {width, length, nstrip};
 
-      if (!theComp11Flag) {
-        //Correction of the orientation to get the REAL geometry.
-        //Change of axes for the +z part only.
-        //Including the 0 whell
-        if (*(tranStart + 2) > -1500.) {  //tran[2] >-1500. ){
-          Basic3DVector<float> newX(-1., 0., 0.);
-          Basic3DVector<float> newY(0., -1., 0.);
-          Basic3DVector<float> newZ(0., 0., 1.);
-          rot.rotateAxes(newX, newY, newZ);
-        }
+      //Correction of the orientation to get the REAL geometry.
```

Comment:
This should also get removed, as it was run if theComp11Flag is false

Reply:
@perrotta Yes, you are right, but I choose to see "in primis" if the problem of the histos comparison will be fixed or not. If yes, I'll follow your suggestion. Tks

Reply:
@perrotta : done, tks again